src/components/CombinedFeaturePanel/CombinedFeaturePanel.tsx

```diff
@@ -79,12 +78,6 @@ export function CombinedFeaturePanel(props: Props) {
             <FeatureSection key={getKey(feature)} feature={feature} />
           ))}
 
-          {(!features || features.length === 0) && (
-            <Callout.Root>
-              <Callout.Text>{t("No features found.")}</Callout.Text>
-            </Callout.Root>
-          )}
-
```

Comment:
I think this has the purpose to show others that there is in fact an error and no features could be found. Removing it will not address the underlying issue, but rather make it harder to debug error in the future. I think we should keep this.